pkgs/tools/audio/beets/default.nix

```diff
@@ -46,13 +57,20 @@ lib.makeExtensible (self: {
   beets-minimal = self.beets.override { disableAllPlugins = true; };
 
   beets-unstable = callPackage ./common.nix {
-    inherit python3Packages extraPatches;
-    version = "2.2.0-unstable-2024-12-02";
+    inherit python3Packages;
+    extraPatches = [];
```

Comment:
preventing the `extraPatches` from above from applying.

❓❓ Should I do this another way ❓❓